imports/state/epics/unverified-warning.js

```diff
@@ -0,0 +1,28 @@
+// import { Accounts } from 'meteor/accounts-base'
+import { Meteor } from 'meteor/meteor'
+import { Subject } from 'rxjs/Subject'
+import { RESEND_VERIFICATION, RESEND_ERROR, RESEND_SUCCESS } from '../../ui/unverified-warning/unverified-warning.actions'
+
+import 'rxjs/add/operator/switchMap'
+import 'rxjs/add/operator/filter'
+
+export const unverifiedWarning = (action$, store) =>
+  action$.ofType(RESEND_VERIFICATION)
+    .filter(() => store.getState().resendVerificationState)
+    .switchMap(({email}) => {
+      const meteorResult$ = new Subject()
+      Meteor.call('resendEmail', Meteor.userId(), email, (err, result) => {
```

Comment:
No need to send the userId, and the email address is passed in although it is never in use by the method.
-> this method should be called with no args at all.